components/script/dom/document.rs

```diff
@@ -4311,6 +4312,12 @@ impl Document {
         self.policy_container.borrow().csp_list.clone()
     }
 
+    pub(crate) fn set_endpoints_list(&self, endpoints: Option<Vec<ReportingEndpoint>>) {
+        if let Some(endpoints) = endpoints {
+            self.window().set_endpoints_list(endpoints);
+        }
+    }
```

Comment:
This is a small thing, but having a function that does nothing if `None` is passed feels like the caller should take care of that.